apps/docs/README.md

```
@@ -0,0 +1,36 @@
+This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://nextjs.org/docs/app/api-reference/cli/create-next-app).
```

Markdownlint (0.37.0)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)